src/document/__tests__/DocumentAnnotator-test.ts

```diff
@@ -286,15 +286,15 @@ describe('DocumentAnnotator', () => {
         });
 
         test('should add and remove is highlighting class if mode changes', () => {
-            expect(annotator.annotatedEl?.classList.contains('ba-is-highlighting')).toBe(false);
+            expect(annotator.annotatedEl?.classList.contains(ANNOTATION_CLASSES[Mode.HIGHLIGHT] || '')).toBe(false);
```

Comment:
Is `|| ''` needed?

Reply:
If `|| ''` is required, can we just use string `'ba-is-create--highlight'` instead?

Reply:
Yes it's required - I think setting `'ba-is-create--highlight' makes more sense than empty string.